platform/install/helm.mdx

```diff
@@ -78,6 +113,56 @@ You can provide multiple values files if desired by specifying additional filena
 Helm has many additional flags, and other ways to pass values, as always, its a great idea to
 check out the Helm [docs](https://helm.sh/docs/) to stay up to date on Helm.
 
+## Login to vCluster Platform
+
+If you haven't setup the domain in `values.yaml`, you can retrieve the domain
+from the secret `loft-router-domain` in the installation namespace.
+
+<Tabs
+  defaultValue="linux"
+  values={[
+    { label: 'Linux and WSL', value: 'linux' },
+    { label: 'macOS', value: 'macos' },
+  ]}
+>
+<TabItem value="linux">
+
+To retrieve the domain from the secret on **Linux**, run:
+
+```bash
+PLATFORM_NAMESPACE=vcluster-platform
+echo "https://$(kubectl get secret loft-router-domain \
+  -n "$PLATFORM_NAMESPACE" \
+  -o jsonpath="{.data.domain}" \
+  | base64 --decode)"
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To retrieve the domain from the secret on **macOS**, run:
+
+```bash
+PLATFORM_NAMESPACE=vcluster-platform
+echo "https://$(kubectl get secret loft-router-domain \
+  -n "$PLATFORM_NAMESPACE" \
+  -o jsonpath="{.data.domain}" \
+  | base64 -D)"
+```
+
+</TabItem>
+</Tabs>
+
+:::info
+Default credentials for testing purposes:
+- Username: admin
+- Password: my-password
+:::
+
+
+Navigate to the domain in your browser and login with the default credentials or
+the credentials you set in values.yaml and complete the profile setup.
+
 ## vCluster Platform Configuration
```

Comment:
Does this make sense at the bottom of the page or should it be higher up before you install? Based on our defaults, it feels like I would deploy and then have to update the config and re-deploy. 

If we keep it at the bottom, I would talk about the config but also link to a section (which might not exist yet) on how to redeploy. 

Something similar to this page that we have in vcluster docs: https://vcluster.com/docs/vcluster/manage/deploy-changes

Reply:
That's a good idea :+1:

Reply:
This is moved now, and the configuration section is in the middle of the flow.